app/src/sharedTest/java/org/simple/clinic/util/RxTestUtils.kt

```diff
@@ -0,0 +1,8 @@
+package org.simple.clinic.util
```

Comment:
Oh, we did it because we were using it in a later commit, but that usage was removed. We still decided to leave this here because this is a better place for this extension method since it can be used in both unit and integration tests.